widgets/text/text.go

```diff
@@ -109,6 +109,17 @@ func (t *Text) Write(text string, wOpts ...WriteOption) error {
 		t.reset()
 	}
 	for _, r := range text {
+		// Implement basic buffer limit to avoid excess memory issues
+		if len(t.content) > 1 && len(t.content) >= t.opts.maxContent && t.opts.maxContent > 0 {
+			// If the new content is longer than all existing, simply reset
```

Comment:
How do we handle the case when a single `text` given to `Write` is longer than the configured `MaxContent`?

Reply:
I've now fixed this case.